src/clj_test_containers/core.clj

```diff
@@ -29,6 +34,24 @@
      :env-vars (.getEnvMap container)
      :host (.getHost container)}))
 
+(defn create-from-docker-file
+  [{:keys [exposed-ports env-vars command docker-file]
+    :or {docker-file "Dockerfile"}}]
+  (let [docker-image (ImageFromDockerfile.)
+        docker-image (-> docker-image
+                         (.withDockerfile (Paths/get "." (into-array [docker-file]))))
+        container (GenericContainer. docker-image)]
+    (.setExposedPorts container (map int exposed-ports))
+    (if env-vars
```

Comment:
The default in the existing code is to always use `some?` for null checks. I would like to use the simple `if` for booleans only, to cut down on confusion